--- origsrc/isoquery-1.7/setup.py	2012-01-23 06:03:14.000000000 -0600
+++ src/isoquery-1.7/setup.py	2012-08-10 05:18:53.485724200 -0500
@@ -69,10 +69,9 @@ class custom_build(build):
         dest = os.path.join("build", "man", "isoquery.1")
         gzip = os.path.join("build", "man", "isoquery.1.gz")
         if newer(source, dest):
-            result = os.system("rst2man %s %s" % (source, dest))
+            result = os.system("rst2man.py %s %s" % (source, dest))
             if result != 0:
                 sys.stderr.write("Fatal error: the manpage could not be generated with rst2man.\n")
-                sys.exit(1)
             result = os.system("gzip --force --best %s" % dest)
             if result != 0:
                 sys.stderr.write("Fatal error: the manpage could not be compressed with gzip.\n")
@@ -101,10 +100,9 @@ class custom_build(build):
                     sys.exit(1)
                 # Generate manpage
                 dest = os.path.join(directory, "isoquery.1")
-                result = os.system("rst2man %s %s" % (translation, dest))
+                result = os.system("rst2man.py %s %s" % (translation, dest))
                 if result != 0:
                     sys.stderr.write("Fatal error: the manpage could not be generated with rst2man.\n")
-                    sys.exit(1)
                 result = os.system("gzip --force --best %s" % dest)
                 if result != 0:
                     sys.stderr.write("Fatal error: the manpage could not be compressed with gzip.\n")
